tests/authorizations/test_email_authorization.py

```diff
@@ -13,9 +13,7 @@ def test_auth_header_is_required(self):
         assert is_valid(response, 401)
         assert response.json == {"message": "Missing authorization header"}
 
-    def test_all_roles_except_admin_are_denied_access(self, auth_headers):
+    def test_all_roles_except_admin_are_denied_access(self, pm_header):
```

Comment:
Can we update the test name. It is saying test all roles, but we are now testing one role. Furthermore, with this new test I'm not sure if join_staff have access or not. With the previous test I knew only admins had access.

Preferably I would like to have 3 tests here. One for each role.